scripts/ibc/setup_ibc.sh

```diff
@@ -95,7 +95,11 @@ echo '--------------------------------- Creating IBC path... -------------------
 
 rly paths new "$ROLLAPP_CHAIN_ID" "$SETTLEMENT_CHAIN_ID" "$RELAYER_PATH" --src-port "$IBC_PORT" --dst-port "$IBC_PORT" --version "$IBC_VERSION"
 
-rly tx link "$RELAYER_PATH" --src-port "$IBC_PORT" --dst-port "$IBC_PORT" --version "$IBC_VERSION"
+if [ "$(rly version -j | jq -r .version)" == "0.3.0-v2.5.2-relayer" ]; then
```

Comment:
we're gonna upgrade the relayer but from the first production relayer and on all relayers need the max-clock-drift so we can remove the `if`